<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>455</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sensor Reading</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,2,2,0" columnminimumwidth="100,0,0,0,0">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" rowspan="5">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="parameterLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="configLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Initialize</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="3" colspan="4">
    <widget class="QStackedWidget" name="monitors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gaugeLayout">
         <property name="spacing">
          <number>3</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="2" column="0" rowspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>SINE FIT</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="durationBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> S</string>
         </property>
         <property name="prefix">
          <string>Duration </string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>DAMPED SINE FIT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="PlotWidget" name="graph"/>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="pauseButton">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5" rowspan="2">
        <widget class="QPushButton" name="saveButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Save Data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" rowspan="2">
        <widget class="QPushButton" name="saveRegionButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Save Region</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QPushButton" name="switcher">
     <property name="text">
      <string>Data Logger</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>initialize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switcher</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>sineFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>dampedSineFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>restartLogging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>pauseLogging(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>saveTraces()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>durationBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog</receiver>
   <slot>setDuration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveRegionButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>saveRegion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeRange(bool)</slot>
  <slot>initialize()</slot>
  <slot>next()</slot>
  <slot>sineFit()</slot>
  <slot>pauseLogging(bool)</slot>
  <slot>dampedSineFit()</slot>
  <slot>restartLogging()</slot>
  <slot>saveTraces()</slot>
  <slot>setDuration()</slot>
  <slot>saveData()</slot>
  <slot>saveRegion()</slot>
 </slots>
</ui>
